composer.json

```diff
@@ -24,7 +24,7 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7 || ^1.0.0",
-        "slevomat/coding-standard": "^8.14",
+        "slevomat/coding-standard": "8.14",
```

Comment:
this is to make CI pass, build failures are resolved in https://github.com/doctrine/coding-standard/pull/325